pypit/core/arextract.py

```diff
@@ -220,7 +1148,7 @@ def boxcar(specobjs, sciframe, varframe, bpix, skyframe, crmask, scitrace, mswav
     # Return
     return bgcorr
 
-
+# This routine is deprecated
```

Comment:
If we really wish to deprecate it, then let's remove it.
Otherwise rename as orig_boxcar.